server/routes/routes.go

```diff
@@ -66,7 +66,7 @@ func Routes(r *gin.Engine, sysInfo SystemInfo, authInfo AuthInfo, baseHref strin
 		// Add the AuthN/AuthZ middleware to the group.
 		r1Group.Use(authMiddleware(authorizer, dexObj))
 	}
-	v1Routes(r1Group)
+	v1Routes(r1Group, dexObj)
```

Comment:
Curious, when auth is disabled, what will be the dexObj? will it have any impact?

Reply:
updated....